README.md

```diff
@@ -2,15 +2,15 @@
 
 A website to track the background simulation in Elite Dangerous
 
-Elite BGS is a project that attempts to fill a gap of APIs for the treasure trove of information online. Currently it is operating in 2 seperate modules.
+Elite BGS is a project that attempts to fill a gap of APIs for the treasure trove of information online. Currently, it is operating in 2 separate modules.
 
 ## EDDB API
 
-Currently [EDDB](https://eddb.io/) is the greatest treasure of information of the galaxy and it is a user friendly tool for many pilots. But it lacks an API for developers. EDDB API is an attempt to fill this gap by providing an API for EDDB data sources.
+Currently, [EDDB](https://eddb.io/) is the greatest treasure of information of the galaxy and it is a user friendly tool for many pilots. But it lacks an API for developers. EDDB API is an attempt to fill this gap by providing an API for EDDB data sources.
 
 ## ELITE BGS API
 
-For ultimate immersion and BGS control, we require not only the latest data but also historical ones. ELITE BGS API is a step in that direction. It stores information which changes regularly in a historical format. Thus, one can always look up the events that led to the current Civil War for instance.
+For ultimate immersion and BGS control, we require not only the latest data but also historical ones. ELITE BGS API is a step in that direction. It stores information which changes regularly in an historical format. Thus, one can always look up the events that led to the current Civil War, for instance.
```

Comment:
I think keeping `a historical` would be better as justified [here](https://en.oxforddictionaries.com/usage/a-historic-event-or-an-historic-event), [here](https://english.stackexchange.com/questions/47090/why-we-say-an-historical-but-a-history) and [here](https://www.alphadictionary.com/articles/drgw007.html)